From 404b9369da7bad8b43cbfcb80800999eace81de4 Mon Sep 17 00:00:00 2001
From: Algea Cao <algea.cao@rock-chips.com>
Date: Fri, 27 Oct 2017 17:08:27 +0800
Subject: [PATCH] ARM: dts: rk3288-evb-android-rk818-mipi: fix up HDMI can't
 display after resume

HDMI can't display afer resume, because HDMI 5v was turn off.
This patch fix up this problem.

Change-Id: Id48ca0616b81ea27416dc733662f388d51b2643c
Signed-off-by: Algea Cao <algea.cao@rock-chips.com>
---
 .../dts/rk3288-evb-android-rk818-mipi.dts     | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
index 2a6d4d797cf9..a174e62e37f2 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
@@ -142,6 +142,8 @@
 		vcc7-supply = <&vcc_sys>;
 		vcc8-supply = <&vcc_sys>;
 		vcc9-supply = <&vcc_io>;
+		boost-supply = <&vcc_sys>;
+		h_5v-supply = <&boost>;
 
 		regulators {
 			vdd_logic: DCDC_REG1 {
@@ -190,6 +192,17 @@
 				};
 			};
 
+			boost: DCDC_BOOST {
+				regulator-always-on;
+				regulator-boot-on;
+				regulator-min-microvolt = <5000000>;
+				regulator-max-microvolt = <5000000>;
+				regulator-name = "boost";
+				regulator-state-mem {
+					regulator-on-in-suspend;
+				};
+			};
+
 			vcca_codec: LDO_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
@@ -302,6 +315,15 @@
 					regulator-on-in-suspend;
 				};
 			};
+
+			h_5v: HDMI_SWITCH {
+				regulator-always-on;
+				regulator-boot-on;
+				regulator-name = "h_5v";
+				regulator-state-mem {
+					regulator-on-in-suspend;
+				};
+			};
 		};
 	};
 };
-- 
2.35.3

